aopy/analysis.py

```diff
@@ -516,6 +516,35 @@ def get_pca_dimensions(data, max_dims=None, VAF=0.9, project_data=False):
 
     return list(explained_variance), num_dims, projected_data
 
+def get_pca_sliding_window(data, max_dims = 10, window_size = 1000):
+    '''
+    Use PCA to estimate the covariance structure
+
```

Comment:
Can you add a little more to this description about how this function uses a sliding window to sequentially calculate PCA.